main.js

```diff
@@ -199,7 +199,6 @@ if (!gotTheLock) {
     mainWindow.on('close', e => {
       if (!shutdownStarted) {
         shutdownStarted = true;
-        childWindow.destroy();
```

Comment:
I'm confused how the `childWindow` ever gets destroyed now.  I see you added `this.windowsService.closeChildWindow()` to the shutdown handler above, but that doesn't actually properly close/destroy the window, it just hides it.  Unless I'm missing something, I don't think the application will cleanly exit now.

Reply:
Do we really need to destroy it? Didn't have any issues with closing the app

Reply:
Electron will not exit until all windows are closed.  Just because all windows are gone that doesn't mean it's not still running in the background.

Reply:
Hmm. The child window is destroying even without this line. But don't see where we can do this.

Reply:
Interesting.  I'm not sure either.

Reply:
Yeah I tested it a bit and it seems fine.  Think we're probably ok here.